### PR TITLE
fix(wallet): in app tx related notifications improvements

### LINKED ISSUE #16338: In app tx related notifications improvements
Improve notifications within the app when the tx is placed/pending/confirmed.

At this moment notifications are a bit intertwined and many of them are missing in the app that we should have by the Figma file:
- https://www.figma.com/design/FkFClTCYKf83RJWoifWgoX/Wallet-v2?node-id=25111-365316&node-type=canvas&m=dev

Signals that need to be revised:
```
const SIGNAL_OWNER_TOKEN_SENT* = "ownerTokenSent"
const SIGNAL_TRANSACTION_SENDING_COMPLETE* = "transactionSendingComplete"
const SIGNAL_TRANSACTION_STATUS_CHANGED* = "transactionStatusChanged"
```

### PR DESCRIPTION
**Pull request by saledjenic**
Corresponding `status-go` PR:
- https://github.com/status-im/status-go/pull/6050

Closes #16338

### CODE REVIEW

**Review by alaibe — APPROVED** [review]

File: ui/app/mainui/AppMain.qml [hunk 1]
```diff
@@ -237,121 +237,276 @@ Item {
...
+                                        error: string) {
+
+            let toastTitle = ""
+            let toastSubtitle = ""
+            let toastIcon = ""
+            let toastLoading = false
+            let toastType = Constants.ephemeralNotificationType.normal
+            let toastLink = ""
+
+            const sender = !!fromName? fromName : SQUtils.Utils.elideText(fromAddr, 6, 4)
```
Comment (caybro) [inline ui/app/mainui/AppMain.qml:269]: Better use `Utils.elideAndFormatWalletAddress()`?

File: ui/app/mainui/AppMain.qml [hunk 2]
```diff
@@ -237,121 +237,276 @@ Item {
...
-                    const toastSubtitle = qsTr("View on %1").arg(networkName)
-                    const urlLink = "%1/%2".arg(appMain.rootStore.getEtherscanTxLink(chainId)).arg(txHash)
-                    let toastType = Constants.ephemeralNotificationType.success
-                    let icon = "checkmark-circle"
-                    if(!success) {
-                        toastTitle = qsTr("Failed to swap %1 to %2 in %3 using %4 on %5").arg(fromSwapAmount).arg(toSwapAmount).arg(fromAccountName).arg(Constants.swap.paraswapHostname).arg(networkName)
-                        toastType = Constants.ephemeralNotificationType.danger
-                        icon = "warning"
+            case Constants.txStatus.failed: {
+                toastTitle = "Send failed: %1 from %2 to %3"
```
Comment (caybro) [inline ui/app/mainui/AppMain.qml:447]: `qsTr()` missing

File: ui/app/mainui/AppMain.qml [hunk 3]
```diff
@@ -237,121 +237,276 @@ Item {
...
+                case Constants.SendType.ERC721Transfer: {
+                    toastTitle = toastTitle.arg(assetName).arg(sender).arg(recipient)
+                    break
+                }
+                case Constants.SendType.ERC1155Transfer: {
+                    toastTitle = qsTr("Send failed: %1 %2 from %3 to %4").arg(fromAmount).arg(assetName).arg(sender).arg(recipient)
+                    break
+                }
+                case Constants.SendType.Swap: {
+                    toastTitle = qsTr("Swapp failed: %1 to %2 in %3").arg(sentAmount).arg(receivedAmount).arg(sender)
```
Comment (caybro) [inline ui/app/mainui/AppMain.qml:488]:
```suggestion
                    toastTitle = qsTr("Swap failed: %1 to %2 in %3").arg(sentAmount).arg(receivedAmount).arg(sender)
```

**Review by caybro — APPROVED** [review]
LGTM :+1:

**Review by virginiabalducci — APPROVED** [review]
These user stories were tested and passed. The failed status was not tested yet. 
Failed status will be tested later on a master build or a specific build where we can emulate tx errors to show the fail

SP-431: notifications for sent asset (erc20 + ETH) to non-saved address
SP-432: notifications for sent asset (erc20 + ETH) to saved address or own address
SP-434: notifications for bridge
SP-435: notifications for swap
SP-436: notifications for spending cap and swap


These missing items will be handled in a different issue https://github.com/status-im/status-desktop/issues/16810
- notifications for WC/BC
- notifications for Community minting and airdropping

**Pull request merged**